spring-boot/admin/src/main/resources/application.yml

```diff
@@ -107,3 +125,11 @@ security:
       userInfoUri: https://api.github.com/user
 
 management.security.enabled: false
+
+# Proxy configuration files are stored in this directory. The value of this parameter needs to direct to the same
+# folder as spring.cloud.config.server.native.searchLocations from the Config Servers application.yml.
+proxy.config.files.folder: ${user.dir}/target/temp-gateway-config-file-storage
+
+# If CoatRack is deployed via IDE/commandline the Gateway config files will be stored in the 'target' folder of the
+# project. If docker-compose is used for deployment, these files will be stored internally within a docker volume.
+is-run-via-docker-compose-setup: false
```

Comment:
This is a default yml config that will be put inside the admin jar file. That jar can be run on its own but it could also be put into a container and be run via docker-compose. 

Therefore I think we simply cannot know here if it is running in a docker-compose setup, so we should not define this parameter here. It could lead to unexpected problems in specific deployment situations.